image_builder/image_builder/api.py

```diff
@@ -63,13 +63,13 @@ def _health():
 def _setup_api_endpoints(api: APIRouter, config: Config, plugin_engine: PluginEngine):
 
     class CredentialsModel(BaseModel):
-        username: str = Field(example="admin")
-        password: str = Field(example="hunter2")
+        username: str = Field(examples=["admin"])
+        password: str = Field(examples=["hunter2"])
```

Comment:
The password needs to be plaintext right? I am only seeing 'password: str = Field(examples=["*******"])'